<?xml version="1.0" ?>

<launch>
    <param name="offset_file_path"      value="$(find test_gripper_manager)/config/offset.yaml"/>
    <param name="robot_file_path"       value="$(find test_gripper_manager)/config/TEST-POWERXEL.robot"/>
    <param name="init_file_path"        value="$(find test_gripper_manager)/config/dxl_init_powerxel_test.yaml"/>
    <param name="loadcell_dev"          value="/dev/loadcell"/>

    <!-- TEST-Gripper Manager -->
    <node name="test_powerxel_manager" pkg="test_gripper_manager" type="test_powerxel_manager" output="screen">
      <param name="robot_name" value="GripperTest"/>
    </node>
</launch>
